run-integration-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+BASE_DIR=`pwd`
+
+# Create Cassandra and Elasticsearch containers to run the integration test
+docker-compose -f contrib/blueflood-docker-compose/docker-compose-integration-test.yml up -d
+
+# Copy Cassandra schema file into Cassandra container
+docker cp src/cassandra/cli/load.cdl cassandra-node:/tmp/load.cdl
```

Comment:
FYI you could volume-attach the file into the container, but not worth changing.